packages/cli/lib/pre-render-data-context.js

```diff
@@ -0,0 +1,26 @@
+import { Component } from 'preact';
+import { createContext } from 'preact-compat';
```

Comment:
this will break X?

Reply:
also force includes compat I guess?

Reply:
Force includes only the context bits of it. Hoping tree shaking is still a thing.

Also I guess this is solvable if we release this as a separate package and ask our users to implement it instead of us doing it magically. 
Or if it sounds like too much work then this should just be a PreactX only feature.

WDYT?

Reply:
no u're right increases the base bundle by 13kb ungzip and 4kb gzip...🙀

Reply:
If we move everything to preact X this delta comes down to 0.3 kb but then every one will be required to be on X and preact 8 wont work